src/main/java/com/minecolonies/util/Utils.java

```diff
@@ -267,10 +284,12 @@ public static int toggleFlag(int data, int flag)
      * @param pos      Coordinates
      * @param block    Block that makes the sound
      * @param metadata Metadata of the block that makes sound
+     * @param citizen  the citizen breaking this block
      */
-    public static void blockBreakSoundAndEffect(@NotNull World world, BlockPos pos, Block block, int metadata)
+    public static void blockBreakSoundAndEffect(@NotNull World world, BlockPos pos, Block block, int metadata, EntityCitizen citizen)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "metadata". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)